smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/endpointsV2/RuleSetParameterFinder.java

```diff
@@ -152,6 +153,64 @@ public Map<String, String> getContextParams(Shape operationInput) {
         return map;
     }
 
+    /**
+     * Get map of params to JavaScript equivalent of provided JMESPath expressions.
+     */
+    public Map<String, String> getOperationContextParamValues(OperationShape operation) {
+        Map<String, String> map = new HashMap<>();
+
+        Optional<OperationContextParamsTrait> trait = operation.getTrait(OperationContextParamsTrait.class);
+        if (trait.isPresent()) {
+            OperationContextParamsTrait operationContextParamsTrait = trait.get();
+            operationContextParamsTrait.getParameters().forEach((name, definition) -> {
+                String separator = ".";
```

Comment:
this doesn't seem to change, it might be easier to read inline

Reply:
Removed in [`a7c13a7` (#1379)](https://github.com/smithy-lang/smithy-typescript/pull/1379/commits/a7c13a7ebcf4a50aff4c8b3fbd9819d0883e1715)

Reply:
I was referring to the separator, but below I see there are reasons for it